### PR TITLE
Convert Collapse accordion from Panels to Cards

### PR DESCRIPTION
**Pull request by Johann-S**
Hi,

Fix for https://github.com/twbs/bootstrap/issues/18375

But with this PR Collapse plugin is now dependent of cards, So if you're ok with that I'll work on how to remove this dependency.

A preview from the result : 

collapse

### CODE REVIEW

**Comment by cvrebert** [comment]
Could you update the Accordion in `js/tests/visual/modal.html` too please?

**Comment by cvrebert** [comment]
> But with this PR Collapse plugin is now dependent of cards, So if you're ok with that

Yeah, that's fine. We can leave removing the dependency to a future pull request.

CC: @mdo for review

**Comment by Johann-S** [comment]
Done for the modal.html collapse.

Thank you for your feedbacks

**Comment by cvrebert** [comment]
Overlaps with #17159.

File: scss/_card.scss [hunk 1]
```diff
@@ -19,6 +19,10 @@
   margin-bottom: $card-spacer-y;
 }
 
+.card-title > a {
```
Comment (cvrebert) [inline scss/_card.scss]: Hmm, in general, we're trying to avoid tag selectors and child selectors.

File: scss/_card.scss [hunk 2]
```diff
@@ -28,6 +32,10 @@
   margin-bottom: 0;
 }
 
+.card-collapse > .card-text {
+  padding: $card-spacer-x;
```
Comment (cvrebert) [inline scss/_card.scss]: Using an `-x` variable for both horizontal and vertical padding isn't right.

**Comment by Johann-S** [comment]
I did changes :+1:

**Comment by Johann-S** [comment]
Since the merge of #17159 by @mdo I rebased my PR.

**Comment by Johann-S** [comment]
This is why I created a class ``.card-collapse`` .

With ``.card-collapse`` : 
collapse_card-collapse

With ``.card-block`` : 
collapse_card-block gif

/CC @cvrebert @mdo

**Comment by mdo** [comment]
Hey @Johann-S, I missed this completely. Can you update this PR to resolve the conflicts? Would love to get this merged :). I'd also ask that you use the changes I made in #20870 so we can avoid the addition of `.card-collapse` entirely.

**Comment by Johann-S** [comment]
Sure I'll work on it !

**Comment by Johann-S** [comment]
Thank you @mdo for what you've done on #20870. 
And it's done I've made the changes 👍

**Comment by mdo** [comment]
Do we need to silence these Hound errors or is there something else we need to change?

**Comment by mdo** [comment]
Oh wait, sorry missed the file that's in—we're working on that elsewhere I think. Cool!

**Comment by Johann-S** [comment]
I think @mdo you forget this PR in the v4 Alpha 5 Ship list (https://github.com/twbs/bootstrap/issues/20630) or I'm wrong ?

**Pull request merged**